sdk/purview/Azure.Analytics.Purview.Account/src/Generated/PurviewCollection.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace Azure.Analytics.Purview.Account
 {
+    // Data plane generated client. The PurviewCollection service client.
```

Comment:
Would it be hard to indicate the difference between service clients and subclients?

Reply:
@annelo-msft No. It's straightforward. I've updated the generator to print `sub-client` if the client is a sub-client. See the latest changes in this PR.

Reply:
Ok, great, thanks @archerzz!